File: 122.BestTimeToBuyAndSellStockTwo/clywin123/maxProfit.py
int maxProfit(int* prices, int pricesSize) {
    int profit = 0,i;
    for(i=1;i<pricesSize;i++){
        if(prices[i]>prices[i-1])
            profit +=(prices[i]-prices[i-1]);
    }
    return profit;
}

File: 121.BestTimeToBuyAndSellStock/wyrj/maxProfit.js
/**
 * @param {number[]} prices
 * @return {number}
 */
var maxProfit = function(prices) {
    let minPrice = prices[0];
    let benefit = 0;
    for (let i = 1; i < prices.length; i++) {
        if (prices[i] < minPrice) minPrice = prices[i];
        else benefit = Math.max(benefit, prices[i] - minPrice);
    }
    return benefit;
};
